server/views/match-defendant.njk

```diff
@@ -1,209 +1 @@
                                                    {% endif %}
                                                    </td>
                                                </tr>
                                                <tr class="govuk-table__row">
                                                    <th scope="row" class="govuk-table__header">CRN</th>
                                                    <td class="govuk-table__cell">{{ item.matchIdentifiers.crn }}</td>
                                                </tr>
                                                <tr class="govuk-table__row">
                                                    <th scope="row" class="govuk-table__header">Probation status</th>
                                                    <td class="govuk-table__cell">{{ item.probationStatus | string | capitalize }}</td>
                                                </tr>
                                                <tr class="govuk-table__row">
                                                    <th scope="row" class="govuk-table__header">Most recent event</th>
                                                    <td class="govuk-table__cell">{{ params.moment(item.mostRecentEvent.startDate, 'YYYY-MM-DD').format('D MMMM YYYY') if item.mostRecentEvent.startDate }}{{ " - " + item.mostRecentEvent.text if item.mostRecentEvent.text }} {{ "(" + item.mostRecentEvent.length + " " + item.mostRecentEvent.lengthUnits + ")" if item.mostRecentEvent.length }}</td>
                                                </tr>
                                                </tbody>
                                            </table>
                                        </div>

                                    </label>
                                </div>
                            {% endfor %}

                        </div>

                    {% endcall %}

                </div>

            </div>
        </div>

        {% from "govuk/components/details/macro.njk" import govukDetails %}
        {%- set detailsText = "<p class=\"govuk-body\">If the defendant is unknown to probation, you must <a href=\"/" + params.courtCode + "/match/defendant/" + params.caseNo + "/nomatch\" class=\"govuk-link govuk-link--no-visited-state\">confirm they have no record</a>.</p><p class=\"govuk-body\">If they have an NDelius record, you can <a href=\"/" + params.courtCode + "/match/defendant/" + params.caseNo + "/manual\" class=\"govuk-link govuk-link--no-visited-state\">link it to them with a case reference number</a>.</p>" -%}
        {{ govukDetails({
            classes: "govuk-!-margin-top-6",
            summaryText: "Can't see the correct record?",
            text: detailsText | safe
        }) }}

        <button type="submit" class="govuk-button">Confirm record</button>

        <p class="govuk-body"><a href="{{ backUrl }}" class="govuk-link govuk-link--no-visited-state">Cancel</a></p>

    </form>

{% endblock %}

```

Comment:
Can you fix the formatting of this file please? Looks like somehow it's now minified or at least all on one line :/